src/mrack/transformers/transformer.py

```diff
@@ -155,3 +162,37 @@ def create_host_requirements(self):
             f"{self.dsp_name} Created {len(reqs)} requirement(s): {object2json(reqs)}"
         )
         return reqs
+
+    def validate_ownership_lifetime(self, host):
+        """Check if ownership and lifetime exists for host."""
+        log_msg_start = f"{self.dsp_name} [{host.get('name')}]"
+        mrack_config = global_context.CONFIG
+        require_owner = mrack_config.require_owner()
+        logger.info(f"owner_info : {require_owner}")
+        if require_owner.lower() == "true":
```

Comment:
required_owner is a boolean (resp. its default value) -> cannot be lowered as string, thus this line fails with:
```
2022-11-04T14:39:56       File "/usr/local/lib/python3.10/site-packages/mrack/transformers/transformer.py", line 172, in validate_ownership_lifetime
2022-11-04T14:39:56         if require_owner.lower() == "true":
2022-11-04T14:39:56     AttributeError: 'bool' object has no attribute 'lower'
```

Reply:
I'd recommend to modify `require_owner` getter in config.py tp ensure that a bool is returned so that consumers cant rely on it being a boolean.

Reply:
Done, please have look in updated code.